### PR TITLE
Add new blog about onboarding employees in CAPP LMS

### PR DESCRIPTION
**Pull request by michellusthof**

### CODE REVIEW

File: source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+author: Bastiaan
+lang: nl
+featured: false
+image: images/blog/social/welcome-1200x675.jpg
+featured_image: images/blog/featured/20191209-welcome.jpg
+header_image: moods/welcome-mood.jpg
+call_to_action: onboarding-contact
+---
+
+__De juiste _onboarding_ van nieuwe medewerkers is van belang om als werkgever aantrekkelijk te blijven en medewerkers aan je te binden. Maar vaak is een onboardingprogramma tijdintensief en brengt het onboarden van medewerkers veel begeleiding (en daarmee hoge kosten) met zich mee. CAPP LMS kan de onboarding niet alleen goed ondersteunen, maar maakt het hele proces ook efficiënter en meetbaar.__
```
Comment (RonaldDijkstra) [inline source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown]: tijdsintensief

File: source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+image: images/blog/social/welcome-1200x675.jpg
+featured_image: images/blog/featured/20191209-welcome.jpg
+header_image: moods/welcome-mood.jpg
+call_to_action: onboarding-contact
+---
+
+__De juiste _onboarding_ van nieuwe medewerkers is van belang om als werkgever aantrekkelijk te zijn en nieuwe medewerkers aan je te binden. Maar vaak is een onboarding/inwerkprogramma tijdsintensief en kost het onboarden van medewerkers veel tijd (en daarmee geld). CAPP LMS verlaagt de uitval van nieuwe medewerkers, verhoogt hun inzetbaarheid en maakt het hele proces efficiënt en meetbaar.__
+
+## Alle facetten van de organisatie
+Tijdens het inwerken wil je medewerkers snel bekend krijgen met alle facetten van de organisatie: de juiste systemen en materialen; snel relaties laten bouwen met collega’s en medewerkers; de organisatie leren kennen op het gebied van visie, missie, cultuur en bedrijfsdoelstellingen.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown]: So we use 'je' in this blog and use 'u' in the CTA?

File: source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Uitval verlagen en inzetbaarheid verhogen
+Met CAPP LMS verlaagt u de uitval van nieuwe medewerkers en verhoogt u hun inzetbaarheid in de organisatie. Ondersteun het fysieke onboarding/inwerkprogramma met digitale middelen - of vervang deze grotendeels - en behaal dezelfde doelstellingen.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown]: Oh and here we use 'u' again.

File: source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Een onboardingprogramma opbouwen  
+Met [CAPP LMS](/capp-lms) bouwt u een onboarding/inwerkprogramma eenvoudig op door de inzet van diverse middelen. Enkele voorbeelden:
+
+-   Maak een enthousiasmerende welkom- en organisatievideo;
+-   Laat medewerkers een quiz over de organisatie maken;
+-   Produceer toegankelijke kennismakingsvlogs per afdeling;
+-   Maak verschillende interactieve artikelen over applicaties waarmee de medewerker gaat werken;
+-   Creëer verschillende events zoals een kennismakingsessie, een teamkennismaking of een oefensessie voor praktijkhandelingen die essentieel zijn voor de uitvoering van het dagelijkse werk van de nieuwe medewerker.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown]: 2 times use of verschillende in two consecutive sentences. Please remove one

File: source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Medewerker zelf in controle
+In het doorlopen van dit programma blijft de medewerker zelf in controle en zelf verantwoordelijk voor het volgen van het programma. Door de onderdelen van het programma te koppelen aan het Kwaliteitspaspoort van [CAPP Compliance](/capp-compliance) heeft de medewerker inzicht in zijn eigen voortgang en monitort de manager tijdens het programma hoe ver de medewerker is en weet exact wanneer deze volwaardig inzetbaar is!
```
Comment (RonaldDijkstra) [inline source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown]: Zelf and zelf. Maybe remove both, but if you want to keep zelf, use the first one.

File: source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+## Alle facetten van de organisatie
+Tijdens het inwerken wil je medewerkers snel bekend krijgen met alle facetten van de organisatie: de juiste systemen en materialen; snel relaties laten bouwen met collega’s en medewerkers; de organisatie leren kennen op het gebied van visie, missie, cultuur en bedrijfsdoelstellingen.
+
+## Uitval verlagen en inzetbaarheid verhogen
+Met CAPP LMS verlaagt u de uitval van nieuwe medewerkers en verhoogt u hun inzetbaarheid in de organisatie. Ondersteun het fysieke onboarding/inwerkprogramma met digitale middelen - of vervang deze grotendeels - en behaal dezelfde doelstellingen.
+
+## Een onboardingprogramma opbouwen  
+Met [CAPP LMS](/capp-lms) bouwt u een onboarding/inwerkprogramma eenvoudig op door de inzet van diverse middelen. Enkele voorbeelden:
+
+-   Maak een enthousiasmerende welkom- en organisatievideo;
```
Comment (RonaldDijkstra) [inline source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown]: Prefer welkomstvideo

File: source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown (hunk 2 above)
Comment (michellusthof) [reply source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown]: You can use 'je' and still address your reader formally. It's more like the Dutch 'men', which is also generic.

Comment (RonaldDijkstra) [reply source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown]: Ah missed that here

Comment (michellusthof) [reply source/blog/nl/2019-12-09-capp-lms-onboarding-nieuwe-medewerkers.html.markdown]: No problem :)

**Pull request merged**